packages/manager/modules/pci/src/projects/new/components/progress/index.html

```diff
@@ -6,6 +6,7 @@
             data-ng-class="{
                 'oui-progress-tracker__step_active': step.active,
             }"
+            data-ng-click="$ctrl.onProgressStepClick(step)"
```

Comment:
**issue(blocking):** We are introducing an accessibility issue here. We recommend to use an element that can be focused.

Reply:
After investigation, for me there is no a11y issue.
If we inspect `li` elements, two attributes are introduced to allow `li` element to got focus:
`role="button"` and `tabindex="0"`.

I think is functional.